src/main/java/com/google/firebase/FirebaseApp.java

```diff
@@ -172,18 +172,30 @@ public static FirebaseApp getInstance(@NonNull String name) {
 
   /**
    * Initializes the default {@link FirebaseApp} instance. Same as {@link
-   * #initializeApp(String)}, but it uses {@link #DEFAULT_APP_NAME} as name.
-   * Also, it uses the {@link FirebaseOptions} values set in the file pointed to by
-   * {@code FIREBASE_CONFIG_ENV_VAR}, if tht file is available.
+   * #initializeApp(String)}, but uses the name {@link #DEFAULT_APP_NAME}.
```

Comment:
Can we change this as follows:

```
Initializes the default {@code FirebaseApp} instance.
Uses Application Default Credentials, and also attempts to load {@code FirebaseOptions}
from the environment. This is done by looking up the {@code FIREBASE_CONFIG} environment
variable. If the value of the variable starts with `{`, it is parsed as a JSON object. 
Otherwise it is treated as a file name,  and the JSON content is read from the corresponding file. 
```